src/scripts/main.js

```diff
@@ -1,3 +1,246 @@
 'use strict';
 
-// write your code here
+const tableBody = document.querySelector('.game-field').firstElementChild;
+const buttonStart = document.querySelector('.start');
+const gameScore = document.querySelector('.game-score');
+const rows = [...tableBody.rows];
```

Comment:
```suggestion
const rows = document.querySelectorAll('.field-row');
```
And then no need to query 'tbody'